documentation/modules/ROOT/pages/index.adoc

```diff
@@ -1,12 +1,36 @@
-= Welcome %USER% to the Container Workshop
```

Comment:
@bfarr-rh Let's keep the whole of this change (sorry, I've not played with the PR merging tools before!)